packages/@ourworldindata/grapher/src/slopeCharts/SlopeChart.tsx

```diff
@@ -324,6 +326,77 @@ export class SlopeChart
         return legendBins.some((bin) => colorsInUse.includes(bin.color))
     }
 
+    @computed
+    private get selectedEntitiesWithoutData(): string[] {
```

Comment:
I'm thinking if it wouldn't be simpler and more straightforward to just do a set difference:
`this.selectedEntityNames - this.series.map(s => s.seriesName)`

What do you think?

Reply:
ohh, that's so much better! I felt so bad for adding these additional transforms, but somehow using `series` didn't occur to me 🤦🏻‍♀️